DTS-STN/Alpha-Site
# content update signup form

# Description

[#476 Change the word "Other" for gender ID question in form to "Another" and Additional Content Changes](https://trello.com/c/noDnCKnX/476-476-change-the-word-other-for-gender-id-question-in-form-to-another-and-additional-content-changes)

Content update for the `gender` question and `visible minority` question. 

**Gender**
<img width="239" alt="Screen Shot 2021-09-23 at 2 31 23 PM" src="https://user-images.githubusercontent.com/64853947/134563781-32b6f2c9-3c23-4e17-bb86-2bee0f9b66eb.png">

<img width="242" alt="Screen Shot 2021-09-23 at 2 30 27 PM" src="https://user-images.githubusercontent.com/64853947/134563794-664aaecb-ce51-4400-a14d-9fc5cc17a9d8.png">

**visible minority**
<img width="238" alt="Screen Shot 2021-09-23 at 2 29 17 PM" src="https://user-images.githubusercontent.com/64853947/134563807-34d0d83b-5277-450f-89d4-d7ac58973984.png">
## Acceptance Criteria

Content matches the design on figma

## Test Instructions

Go to signup page and check the gender question and the visible minority question in both English and French.
